portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,86 +37,90 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-    
-  private ArrayList<String> jsonComments = new ArrayList<String>();
-
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    
-    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
-    
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    PreparedQuery results = datastore.prepare(query);
-    
-    int numComments = getNumComments(request);
-    ArrayList<Comment> comments = new ArrayList<>();
-    for (Entity entity : results.asIterable()) {
-      long id = entity.getKey().getId();
-      String commentBody = (String) entity.getProperty("body");
-      long timestamp= (long) entity.getProperty("timestamp");
-      Comment comment = new Comment(id, commentBody, timestamp);
-      if (numComments > 0) {
-        comments.add(comment);
-        numComments--;
-      }
-    }
-    Gson gson = new Gson();
-
-    response.setContentType("application/json;");
-    response.getWriter().println(gson.toJson(comments));
-  }
-
-  /**
+
+	@Override
+	public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+		Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+
+		DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+		PreparedQuery results = datastore.prepare(query);
+
+		int numComments = getNumComments(request);
+		String langComments = request.getParameter("langComments");
```

Comment:
what does langComments mean? maybe pick a clearer variable name or add a comment?

Reply:
changed the langComments variable to languageForComments to make it clearer